roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -207,6 +207,10 @@ kube_profiling: false
 # Container for runtime
 container_manager: docker
 
+# Enable Kata Containers as additional container runtime
+# When enebled, it requires container_manager=containerd
```

Comment:
typo with 'enebled' instead of 'enabled'

Reply:
Fixed!